RESTserver/src/main/java/com/softserve/if072/restservice/repository/HistoryRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.softserve.if072.restservice.repository;
+
+import com.softserve.if072.common.model.History;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Date;
+
+/**
+ * Created by Nazar Vynnyk
+ */
+
+@Repository
+public interface HistoryRepository extends JpaRepository<History, Integer> {
+
+    Page<History> findAll(Pageable pageable);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)